packit_service/worker/handlers/koji.py

```diff
@@ -354,3 +366,79 @@ def run(self):
         self.build.set_build_logs_urls(koji_build_logs)
 
         return TaskResults(success=True, details={"msg": msg})
+
+
+@configured_as(job_type=JobType.koji_build_tag)
+@reacts_to(event=KojiBuildTagEvent)
+class KojiBuildTagHandler(
+    JobHandler, ConfigFromEventMixin, PackitAPIWithDownstreamMixin
+):
+    task_name = TaskName.koji_build_tag
+
+    _koji_helper: Optional[KojiHelper] = None
+
+    @property
+    def koji_helper(self):
+        if not self._koji_helper:
+            self._koji_helper = KojiHelper()
+        return self._koji_helper
+
+    @staticmethod
+    def get_checkers() -> Tuple[Type[Checker], ...]:
+        return (SidetagExists,)
+
+    def run(self) -> TaskResults:
+        dg_base_url = getenv("DISTGIT_URL", DISTGIT_INSTANCES["fedpkg"].url)
+        sidetag = SidetagModel.get_by_koji_name(self.data.tag_name)
+        sidetag_group = sidetag.sidetag_group.name
+        builds = self.koji_helper.get_builds_in_tag(self.data.tag_name)
+        tagged_packages = {b["package_name"] for b in builds}
+        logger.debug(f"Packages tagged into {self.data.tag_name}: {tagged_packages}")
+
+        packages_to_trigger = set()
+        for package_name in tagged_packages:
+            distgit_project_url = f"{dg_base_url}rpms/{package_name}"
+            project = self.service_config.get_project(url=distgit_project_url)
+            packages_config = PackageConfigGetter.get_package_config_from_repo(
+                base_project=None,
+                project=project,
+                pr_id=None,
+                reference=None,
+                fail_when_missing=True,
```

Comment:
do we want to have this `True`? maybe we should handle this case in a different way?

Reply:
Right, I didn't realize it's possible for builds of packages without Packit config to be tagged into the sidetag, so in this case skipping the package would be certainly better, same below. Let me change that.